docs/Developers/Writing_Good_Commits.rst

```diff
@@ -71,7 +71,11 @@ Example:
 
     setup.py: Change bears' entrypoint
 
--  Maximum of 50 characters.
+-  Maximum of 50 characters. 50 characters `is` a hard limit
```

Comment:
```diff
+   50 characters `is` a hard limit
```

That just repeats "Maximum of 50 characters."  It isnt necessary, and it detracts from the clear and simple "Maximum of 50 characters.". "less is more"  What I wrote in the last review was to make it clear that it isnt just a recommendation, or good idea, which can be ignored. 

```diff
+   (The bot will block any PR which has commits with 51 characters
+   or more).
```

This does add more information, but should be done as a proper sentence, and not using unexplained terms like 'PR'.